libs/cms/src/lib/cms.contentful.service.ts

```diff
@@ -216,6 +216,7 @@ export class CmsContentfulService {
 
   async getOrganizationTitles(
     organizationKeys: string[],
+    locale?: 'en' | 'is',
```

Comment:
Isn't there a better type that could be used here instead of 'en' | 'is' ?

Reply:
Já mætti alveg vera type Locale = 'is' | 'en'